interfaces/builtin/fwupd.go

```diff
@@ -388,9 +388,16 @@ func (iface *fwupdInterface) UDevPermanentSlot(spec *udev.Specification, slot *s
 	if !implicitSystemPermanentSlot(slot) {
 		spec.TagDevice(`KERNEL=="drm_dp_aux[0-9]*"`)
 		spec.TagDevice(`KERNEL=="gpiochip[0-9]*"`)
-		spec.TagDevice(`KERNEL=="tpm[0-9]*"`)
-		spec.TagDevice(`KERNEL=="nvme[0-9]*"`)
+		spec.TagDevice(`KERNEL=="i2c-[0-9]*"`)
+		spec.TagDevice(`KERNEL=="ipmi[0-9]*"`)
 		spec.TagDevice(`KERNEL=="mei[0-9]*"`)
+		spec.TagDevice(`KERNEL=="mtd[0-9]*"`)
+		spec.TagDevice(`KERNEL=="nvme[0-9]*"`)
+		spec.TagDevice(`KERNEL=="tpm[0-9]*"`)
+		spec.TagDevice(`KERNEL=="tpmrm[0-9]*"`)
```

Comment:
TPM resource manager is new thing here, IDK if it needs an ack from @alexmurray

Reply:
Yes, I had marked as "Needs security review". I have requested is review now.

This is new for the cgroup, but it was already the apparmor profile.

Reply:
tpmrm doesn't provide any additional risk compared to directly accessing tpm devices so there is no security risk here.